contrib/NeonVM/stack.c

```diff
@@ -77,16 +77,73 @@ Cell *top(TStack *stack)
 
 Cell *peek(TStack *stack, int element)
 {
-    if (isEmpty(stack)) {
+    if (isEmpty(stack) || ((stack->top - element) < 0)) {

```

Comment:
That's not what I meant. My point was that if the stack is empty, then `stack->top - element` is going to be less than zero anyway, so all you need is:

    if (stack->top - element < 0)

Or, simplifying that further:

    if (element > stack->top)

Reply:
Changed.